modal/cli/run.py

```diff
@@ -290,6 +291,11 @@ def deploy(
 
     deploy_app(app, name=name, environment_name=env, public=public)
 
+    if stream_logs:
+        from modal.cli.app import logs
+
+        logs("", name=name, env=env)
+
```

Comment:
Because `deploy_app` returns an object with the `app_id`, I think we can avoid coupling the CLI endpoints (and the name-to-ID RPC in `logs`) by using the same underlying helper function as `logs()` directly:

```
       try:
            with output_mgr.show_status_spinner():
                await get_app_logs_loop(app_id, client, output_mgr)
        except asyncio.CancelledError:
            pass
```